synapse/handlers/federation_event.py

```diff
@@ -1314,19 +1324,22 @@ async def _handle_marker_event(self, origin: str, marker_event: EventBase) -> No
             marker_event,
         )
 
-    async def backfill_event(
+    async def backfill_event_id(
         self, destination: str, room_id: str, event_id: str
-    ) -> None:
+    ) -> EventBase:
         """Backfill a single event and persist it as a non-outlier which means
         we also pull in all of the state and auth events necessary for it.
 
         Args:
             destination: The homeserver to pull the given event_id from.
             room_id: The room where the event is from.
             event_id: The event ID to backfill.
+
+        Raises:
+            FederationError if we are unable to find the event from the destination
         """
         logger.info(
-            "backfill_event event_id=%s from destination=%s", event_id, destination
+            "backfill_event_id: event_id=%s from destination=%s", event_id, destination
```

Comment:
this has got mangled I think

Reply:
It is outdated now but still don't know what this was about.

Reply:
yup, no idea what was going on here. I'm going to blame github for showing me a mangled diff.